pkg/system/templates/etc/systemd/system/kubelet.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Description=kubelet: The Kubernetes Node Agent
+Documentation=http://kubernetes.io/docs/
+After=docker.service
+Requires=docker.service
+
+[Service]
+ExecStart=/usr/bin/kubelet \
+  --address=0.0.0.0 \
+  --authentication-token-webhook \
+  --authorization-mode=Webhook \
+  --allow-privileged=true \
+  --cloud-provider=aws \
+  --cluster-domain=cluster.local \
+  --cni-bin-dir=/opt/cni/bin \
+  --cni-conf-dir=/etc/cni/net.d \
+  --container-runtime=docker \
+  --network-plugin=cni \
+  --cgroup-driver=cgroupfs \
+  --register-node=true \
+  --kubeconfig=/var/lib/kubelet/kubeconfig \
+  --feature-gates=RotateKubeletServerCertificate=true \
+  --anonymous-auth=false \
+  --client-ca-file=/etc/kubernetes/pki/ca.crt $KUBELET_ARGS $KUBELET_MAX_PODS $KUBELET_EXTRA_ARGS
```

Comment:
`$KUBELET_EXTRA_ARGS` doesn't seem to be populate. Is this for additional user-provided configs? Should we document it?

Reply:
Now that I re-read AWS's boostrap.sh, I think we're missing a `/etc/systemd/system/kubelet.service.d/30-kubelet-extra-args.conf` template file.

Reply:
Yeah lets skip this for now ... until we provide some mechanism to provide the args to ekstrap ...

Reply:
It doesn't hurt to leave this here actually ... it is quite possible for someone to add there own drop in if they need to pass some extra args here ... there is no harm in this variable being here if it is not set though ...

Reply:
Should we drop a small note about it on the README then?